pkg/asset/installconfig/kubevirt/utils.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubevirt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+var (
+	kubeConfigEnvName         = "KUBECONFIG"
+	kubeConfigDefaultFilename = filepath.Join(os.Getenv("HOME"), ".kube", "config")
+)
+
+// LoadKubeConfigContent returns the kubeconfig file content
+func LoadKubeConfigContent() ([]byte, error) {
+	kubeConfigFilename := os.Getenv(kubeConfigEnvName)
+	// Fallback to default kubeconfig file location if no env variable set
+	if kubeConfigFilename == "" {
+		kubeConfigFilename = kubeConfigDefaultFilename
+	}
+
+	return ioutil.ReadFile(kubeConfigFilename)
+}
+
+// WaitForDeletionComplete waits until the resource is not found, getFunc returns error
+func WaitForDeletionComplete(name string, getFunc func() error) error {
+	var (
+		getErr  error
+		counter int32
+	)
+	retries := int32(10)
+	for ; getErr == nil; getErr = getFunc() {
+		if counter == retries {
+			return fmt.Errorf("the resource %s is failed to be deleted (timeout)", name)
+		}
+		counter++
+		time.Sleep(time.Duration(rand.Int31n(counter)) * time.Second)
+	}
```

Comment:
Use wait.ExponentialBackoff.
https://github.com/kubernetes/apimachinery/blob/master/pkg/util/wait/wait.go#L387-L397

Reply:
I removed this function
I changed destroyer to use wait.ExponentialBackoff instead